app/views/posts/index.html.erb

```diff
@@ -1,41 +1,11 @@
-<div class="users-list">
- <div class="single-user">
-<div class="user-image"></div>
-<div class="user-data">
-<div class="user-username"><%= @posts[0].user_name %></div>
-<div class="user-posts">number of posts: 
-<% if @posts[0].user_posts_counter.to_i > 0 %>
-  <%= @posts[0].user_posts_counter %>
-<% else %>
-  0
+<%= render partial:'partials/user_partial', locals: {user: @user}%>
+<% @user_posts.each do |post| %>
+<%= render partial:'partials/post_partial', locals: {post: post}%>
+<ul>
+<% post.recent_comments.each do |comment|%>
+<li> <%= comment.author.name%>: <%= comment.text%></li>
```

Comment:
- [ ] Great job woking on it but please make sure to have the same wireframe as I asked previously  it would be great if you make you layout the same as the given wireframe no styles at this moment but the wireframe it should be the same so kindly work on it to have it similar to the given one 🔥❤🚀


- [ ] I think I can't see the posts even I am sure that I entered some posts in rails c but I can't see them so could please make sure to add the required view to display some posts it is like this for me ⬇⬇
-  this is how I entered it
![image](https://user-images.githubusercontent.com/58553711/231932330-e3c845a6-f0c5-4c95-ae61-e041e3d3adc8.png)


- yours
![image](https://user-images.githubusercontent.com/58553711/231931636-b39a18eb-f1cc-4828-83e0-f98d9604f88c.png)

- It should looks like
![image](https://user-images.githubusercontent.com/58553711/231931741-a544ef14-72f2-42b8-aa23-01e943dc9707.png)

Reply:
@Reem-lab Fix the style issue and adapted the wireframe